### PR TITLE
[v0.14.0.x] bump version to 0.14.0.1 and prepare release notes

### PR DESCRIPTION
**Pull request by codablock**

### CODE REVIEW

File: doc/man/dash-cli.1 [hunk 1]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.10.
-.TH DASH-CLI "1" "May 2019" "dash-cli v0.14.0.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH DASH-CLI "1" "May 2019" "dash-cli v0.14.0.1" "User Commands"
 .SH NAME
-dash-cli \- manual page for dash-cli v0.14.0.0
+dash-cli \- manual page for dash-cli v0.14.0.1
 .SH DESCRIPTION
-Dash Core RPC client version v0.14.0.0
+Dash Core RPC client version v0.14.0.1\-dirty
```
Comment (UdjinM6) [inline doc/man/dash-cli.1]:
```suggestion
Dash Core RPC client version v0.14.0.1
```

File: doc/man/dash-qt.1 [hunk 2]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.10.
-.TH DASH-QT "1" "May 2019" "dash-qt v0.14.0.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH DASH-QT "1" "May 2019" "dash-qt v0.14.0.1" "User Commands"
 .SH NAME
-dash-qt \- manual page for dash-qt v0.14.0.0
+dash-qt \- manual page for dash-qt v0.14.0.1
 .SH DESCRIPTION
-Dash Core version v0.14.0.0 (64\-bit)
+Dash Core version v0.14.0.1\-dirty (64\-bit)
```
Comment (UdjinM6) [inline doc/man/dash-qt.1]:
```suggestion
Dash Core version v0.14.0.1 (64\-bit)
```

File: doc/man/dash-tx.1 [hunk 3]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.10.
-.TH DASH-TX "1" "May 2019" "dash-tx v0.14.0.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH DASH-TX "1" "May 2019" "dash-tx v0.14.0.1" "User Commands"
 .SH NAME
-dash-tx \- manual page for dash-tx v0.14.0.0
+dash-tx \- manual page for dash-tx v0.14.0.1
 .SH DESCRIPTION
-Dash Core dash\-tx utility version v0.14.0.0
+Dash Core dash\-tx utility version v0.14.0.1\-dirty
```
Comment (UdjinM6) [inline doc/man/dash-tx.1]:
```suggestion
Dash Core dash\-tx utility version v0.14.0.1
```

File: doc/man/dashd.1 [hunk 4]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.10.
-.TH DASHD "1" "May 2019" "dashd v0.14.0.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH DASHD "1" "May 2019" "dashd v0.14.0.1" "User Commands"
 .SH NAME
-dashd \- manual page for dashd v0.14.0.0
+dashd \- manual page for dashd v0.14.0.1
 .SH DESCRIPTION
-Dash Core Daemon version v0.14.0.0
+Dash Core Daemon version v0.14.0.1\-dirty
```
Comment (UdjinM6) [inline doc/man/dashd.1]:
```suggestion
Dash Core Daemon version v0.14.0.1
```

**Comment by codablock** [comment]
fyi, test failures are timeouts while uploading cache or completely unrelated to this PR.

**Pull request merged**